axonserver/src/main/java/io/axoniq/axonserver/localstorage/TrackingEventProcessorManager.java

```diff
@@ -200,9 +208,15 @@ private int sendNext() {
                         && count < MAX_EVENTS_PER_RUN
                         && eventIterator.hasNext()
                 ) {
-                    eventStream.onNext(eventIterator.next().asInputStream());
-                    if (permits.decrementAndGet() == 0) {
-                        lastPermitTimestamp.set(System.currentTimeMillis());
+                    SerializedEventWithToken next = eventIterator.next();
+                    if( !blacklisted(next)) {
```

Comment:
Super nit: the default space between `if` and `(` should/could be covered by the style guide I think.